setup.c

```diff
@@ -42,7 +42,7 @@ void setup(void)
 
 	// enable RNG
 	rcc_periph_clock_enable(RCC_RNG);
-	RNG_CR |= RNG_CR_IE | RNG_CR_RNGEN;
+	RNG_CR |= RNG_CR_RNGEN;
```

Comment:
Good point. I'm sure that disabling the ENG_CR_IE does not affect whether SEIS will be set (or not). Not setting / unsetting this bit is just for completeness.

But it makes me wonder if we need to totally reset the RNG peripheral due to the order of operations in 20.3.1 where the interrupt is supposed to be enabled before enabling the peripheral, if you're going to use the RNG interrupt. I would guess it's probably safe to just unset RNG_CR_IE in setupApp though since we don't handle any pending RNG interrupts anyways.